algorithms/bit_manipulation/Sum_vs_XOR.java

```diff
@@ -0,0 +1,23 @@
+//Problem: https://www.hackerrank.com/challenges/sum-vs-xor
```

Comment:
Nice job!
Please, change the name to lowercase to keep the standard of other solutions.
A nice name would be `sum_vs_xor.java`